Machine_Learning/Basic_Projects_of_Machine_Learning/Breast Cancer Classification/requirements.txt

```diff
@@ -0,0 +1,7 @@
+The required libraries for this project work
+
+Numpy
```

Comment:
Include the versions of the libraries used here.

Reply:
Sir I have made the changes sir can you please verify it.